galata/test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json

```diff
@@ -31,6 +31,14 @@
       "Ctrl Shift ,"
     ]
   },
+  {
+    "id": "application:toggle-side-tabbar",
+    "label": "Toggle Sidebar Element",
+    "caption": "By default opens first element of the sidebar",
```

Comment:
```suggestion
```
This one should also be removed, then we should be good to merge.

Reply:
Removed